UserStudentApp/serializers.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import serializers
+from .models import Student, CustomUser, Course, Image
+
+class UserSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
There should always be 2 blank line before starting a new class. (PEP 8) 
Please follow it through out the project.

Reply:
Okay sir, I will take care of the code writing standards